README.md

```diff
@@ -20,8 +20,4 @@ This is a very basic demonstration of using Corda to transfer files from one nod
 9. The file will disappear from that directory / node and appear in another directory being transferred via the Corda network that you are running. The destination directory will appear under build/DATETIMEOFBUILD/CorpA/incoming_1
 
```

Comment:
- You mention in "Background" that it can be run via the command line. If so, you should say how
- Full stop at the end of "Instructions" title - inconsistent
- Provide full path to `Senderkt`